### PR TITLE
change timeout for proxy postsubmit for master to 2 hours

### PR DESCRIPTION
**Pull request by rkpagadala**
Proxy post submit for master needs more time, change timeout to 2 hours for master branch
As per request from @qiwzhang

### CODE REVIEW

File: prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml [hunk 1]
```diff
@@ -64,4 +84,4 @@ postsubmits:
     labels:
       preset-service-account: "true"
     spec:
-      <<: *bazel_spec
+      <<: *bazel_postsubmit_spec
```
Comment (hklai) [inline prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml:87]: I think you can reuse the same stanza and override only the timeout.

e.g. https://github.com/istio/test-infra/blob/master/prow/config.yaml#L75

Comment (rkpagadala) [reply prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml:87]: That one does not override anything Andy, it just adds master and release-1.1 in addition to preexisting config

**Comment by hklai** [comment]
master used to be in the stanza above and it got overridden. I think it was
a problem I fixed a while back.

Have you tried it here?

File: prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml (hunk 1 above)
Comment (sebastienvas) [reply prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml:87]: you can only overwrite with a map, not a list. Krishna is right.

**Pull request merged**